rest/taskana-rest-spring/src/main/java/pro/taskana/classification/rest/ClassificationController.java

```diff
@@ -48,56 +49,33 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(ClassificationController.class);
 
   private static final String LIKE = "%";
-
   private static final String NAME = "name";
-
   private static final String NAME_LIKE = "name-like";
-
   private static final String KEY = "key";
-
   private static final String DOMAIN = "domain";
-
   private static final String CATEGORY = "category";
-
   private static final String TYPE = "type";
-
   private static final String CUSTOM_1_LIKE = "custom-1-like";
-
   private static final String CUSTOM_2_LIKE = "custom-2-like";
-
   private static final String CUSTOM_3_LIKE = "custom-3-like";
-
   private static final String CUSTOM_4_LIKE = "custom-4-like";
-
   private static final String CUSTOM_5_LIKE = "custom-5-like";
-
   private static final String CUSTOM_6_LIKE = "custom-6-like";
-
   private static final String CUSTOM_7_LIKE = "custom-7-like";
-
   private static final String CUSTOM_8_LIKE = "custom-8-like";
 
-  private static final String SORT_BY = "sort-by";
-
-  private static final String SORT_DIRECTION = "order";
-
   private final ClassificationService classificationService;
+  private final ClassificationRepresentationModelAssembler modelAssembler;
```

Comment:
Do we have a rule in the coding guideline with respect to naming variables?

Reply:
cop?